desktop/apps/consignments2/components/thank_you/index.js

```diff
@@ -0,0 +1,73 @@
+import Document from '../../../../components/main_layout/public/icons/consignments-doc.svg'
+import GreenCheck from '../../../../components/main_layout/public/icons/green-check.svg'
+import OpposingArrows from '../../../../components/main_layout/public/icons/consignments-opposing-arrows.svg'
+import PropTypes from 'prop-types'
+import React from 'react'
+import SpeechBubble from '../../../../components/main_layout/public/icons/consignments-speech-bubble.svg'
+import { connect } from 'react-redux'
+
+function ThankYou ({ submission }) {
+  const submissionImage = submission.image_url ? submission.image_url : '/images/missing_image.png'
+
+  return (
+    <div className='consignments2-submission-thank-you'>
```

Comment:
No preference -- I wish there was something more elegant than having to call functions in `className={b()}` but i think thats more literal and reliable than having a "magic" higher order component that patches things for us. That said I haven't spent much time looking (was mostly using CSS Modules before Artsy) and there might be something out there that's nice.